src/build_log_test.cc

```diff
@@ -63,9 +63,9 @@ TEST_F(BuildLogTest, WriteRead) {
 
   ASSERT_EQ(2u, log1.entries().size());
   ASSERT_EQ(2u, log2.entries().size());
-  BuildLog::LogEntry* e1 = log1.LookupByOutput("out");
+  BuildLog::LogEntryPtr e1 = log1.LookupByOutput("out");
```

Comment:
Take the return value by const& to prevent incrementing the refcount unnecessarily.